kw

```diff
@@ -33,11 +33,15 @@ export KWORKFLOW
 # The include function (sourced from this file) should always be used for file sourcing.
 . "$KW_LIB_DIR/kw_include.sh" --source-only
 
+include "$KW_LIB_DIR"/signal_manager.sh
+
 function kw()
 {
   action="$1"
   shift
 
+  signal_manager
```

Comment:
Since `signal_manager` can return 22, you need to handle it here and in the `mk.sh` file. Just add a warning, in this case, we don't anything else.